### DISCUSSION TITLE
When does SoftFever branch merge into main branch?

### DISCUSSION THREAD

**Discussion by TonyK54** [OrcaSlicer/OrcaSlicer#855] [General]
@SoftFever , when does the SoftFever branch merge back into the main branch? If not, should I do a new pull request for the main branch?  Thank you,

**Comment by SoftFever** [comment]
@TonyK54 
I didn't pay attention that the PR went to softfever branch.
I will merge it into the main.
In the future, we can create PR for the main branch directly.

**Reply by TonyK54** [reply]
Thank you. I missed my own mistake. I can do a new pull request to main if you would like. I have it queued up and just need to test locally.
